### DISCUSSION TITLE
Multi database support

### DISCUSSION THREAD

**Discussion by akash-mahmud** [prisma/prisma#20882] [Q&A]
### Question

I have 3 databases in my project. Every database has different data. But they have some relation between them. For example on the transaction database in my transaction, it has a userId stored and in the Default database User table has this user.

Now how to manage these multi-database and cross-relation between databases in Prisma?

**Comment by nurul3101** [comment]
Hello @akash-mahmud 👋 

You would need to create 3 schema files and 3 PrismaClient Instances, one for each of your database.

Please check this GitHub Issue for details:

- https://github.com/prisma/prisma/issues/2443#issuecomment-630679118

Regarding the cross-relation between databases, prisma currently does not support foreign key relations across different databases. You'll have to manage these relations in your application logic.

It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 
Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
